gap/PolygonalComplexes/incidence_geometry.gi

```diff
@@ -586,8 +586,69 @@ AddPropertyIncidence( SIMPLICIAL_ATTRIBUTE_SCHEDULER,
     ["VerticesAttributeOfComplex", "EdgesOfVertices", "EdgesOfFaces", 
         "FacesOfEdges", "VerticesOfEdges", "RamifiedEdges"], ["IsPolygonalComplex"]);
 
+
+InstallMethod(StarNC, "for a polygonal surface and an integer", [IsPolygonalSurface, IsInt],
+	function(surface, vertex)
+	
+		return Star(surface)[vertex];
```

Comment:
Maybe instead of calling Star(surface) we can do it separately here, or we would do a lot of companions/storing for big surfaces